raiden/transfer/node.py

```diff
@@ -510,7 +510,7 @@ def handle_block(chain_state: ChainState, state_change: Block) -> TransitionResu
     return TransitionResult(chain_state, events)
 
 
-def handle_chain_init(
+def handle_action_chain_init(
```

Comment:
nitpick question: shouldn't it be `handle_action_init_chain` ?

Reply:
Good point, I missed that.